web/src/message_list_tooltips.js

```diff
@@ -139,6 +137,25 @@ export function initialize() {
         },
     });
 
+    message_list_tooltip(".message_reactions .reaction_button", {
+        delay: LONG_HOVER_DELAY,
+        placement: "bottom",
+        onShow(instance) {
+            if (!document.body.contains(instance.reference)) {
+                // It is possible for the single reaction necessary for
+                // displaying the reaction button to be removed before
+                // `onShow` is triggered, so we check if the element
+                // exists before proceeding. Because of the longer delay,
+                // this is the only removal check
+                return false;
+            }
+            return true;
+        },
+        onHidden(instance) {
+            instance.destroy();
+        },
+    });
```

Comment:
@amanagr does this the same observer logic that we had for actual emoji reactions, above?

Reply:
3e2b6b52d834fa223dac9dced21e348e329d42c3 is the context. My guess is the answer is yes.